addons/html_editor/static/src/core/history_plugin.js

```diff
@@ -654,6 +660,9 @@ export class HistoryPlugin extends Plugin {
             this.revertMutations(stepToRevert.mutations);
         }
         this.applyMutations(newStep.mutations);
+        this.dispatch("NORMALIZE", {
+            node: this.getMutationsRoot(newStep.mutations) || this.editable,
+        });
```

Comment:
Why do you need it ?

Reply:
It's related to my comment above:
> @Mtaylorr Hello! For this to work well in collaboration, we filter contenteditable attribute changes in filterMutationRecords, to avoid sharing the change in collaboration.
> 
> But by doing so, we also should add dispatch("NORMALIZE") before the dispatch("ADD_EXTERNAL_STEP") in addExternalStep of the history_plugin, because since contenteditable attribute won't be observed, changes to it won't be observed in collaboration when they need to, so the nodes should be normalized on arrival, so that the attribute is applied again by concerned plugins